syntax = "proto3";

package v2.response;

import "v2/page.proto";
import "v2/error.proto";

option go_package = "github.com/hotstar/hs-core-api-go/v2/response";
option java_multiple_files = true;
option java_package = "com.hotstar.bff.api.v2.response";

message PageResponse {
    Success success = 1;
    message Success {
        Page page = 1;
    }
    
    Error error = 2;
}
